specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/ExpressRoutePortAuthorizationDelete.json

```diff
@@ -8,7 +8,11 @@
   },
   "responses": {
     "200": {},
-    "202": {},
+    "202": {
+      "headers": {
+        "Azure-AsyncOperation": "<azure-operation-result>"
```

Comment:
><azure-operation-result> [](http://example.com/codeflow?start=33&length=24)

please provide a sample url , so that it gives an idea of what the header value looks like.